### PR TITLE
Created the Logo file and fixed a null pointer exception w.r.t World

### PR DESCRIPTION
**Pull request by Raveesh**
Created  a log file from [Ascii2Text](http://patorjk.com/software/taag/#p=display&f=Epic&t=WORLD%20%20WAR%20%20OF%20%20ANTS) 

World reference was not initialized, which was throwing a NPE. Initialized it in the constructor

### CODE REVIEW

File: src/main/java/Main.java [hunk 1]
```diff
@@ -1,14 +1,29 @@
...
+        printLogo();
         Game game = new Game(4);
 
         while (!game.isFinished()) {
             game.tick();
             // Render goes here
         }
     }
+
+    private static void printLogo() {
```
Comment (alextopher0) [inline src/main/java/Main.java]: If you wouldn't mind leaving this method out, team-echo is working on a larger scale implementation of printing to the console :)

Comment (Raveesh) [reply src/main/java/Main.java]: Got it. Shall I create a new PR?

Comment (alextopher0) [reply src/main/java/Main.java]: Nope just commit and push changes to your fork. Github will update this PR for you

Comment (alextopher0) [reply src/main/java/Main.java]: Alright I'm gonna check this out before we I get someone else to merge it

Comment (EhWhoAmI) [reply src/main/java/Main.java]: I was thinking the world object could be initialized outside, when we have to load games later.

**Comment by Veradux** [comment]
This seems outdated as well. If that's the case, do you mind closing it

**Comment by alexdqmf** [comment]
Like Vera said, this one is outdated. I also say we close it.

**Comment by Plat251** [comment]
This is indeed outdated.
Also, "created a logo" and "fixed a null pointer exception" are not in the same ball park at all - please don't put them in the same branch, or Pull Request, in the future.

**Pull request closed without merging**